app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -106,6 +107,10 @@ object DisplayWidth extends Enumeration {
   implicit val displayWidthWrites: Writes[DisplayWidth] = Writes.enumNameWrites
 }
 
+object TextAreaRows {
```

Comment:
I think we can keep this object name as TextArea, no need to define another object just for it

`
object TextArea {
   val default = 5
}
`

Reply:
Done.